backend/rebel_hubs/urls.py

```diff
@@ -23,4 +23,5 @@
     path('api/', include('Posts.urls')),
     # path('api/', include('Comments.urls')), will be added in the next PR
     path('', include('calendar_app.urls')),
+    path('', include('users.urls')),
```

Comment:
ideally we would add the api/ right here in order for your urls in users.urls to look a lot cleaner, it's just preference but if you look at how Posts does it it seems better

Reply:
it would look something like `path('api/', include('users.urls')),` and you can remove your api/ prefix in your users.url but again thats totally up to you im just glad we are following a convention

Reply:
Im not sure that code even works this was the result 
<img width="551" alt="api" src="https://github.com/user-attachments/assets/e14d1180-f8aa-45f0-b242-c8b2141215bb">
<img width="442" alt="urls" src="https://github.com/user-attachments/assets/d93b41e3-6f56-486e-8847-ffa9f3af243f">

Reply:
i went to test it out for myself and it works.
![image](https://github.com/user-attachments/assets/70e9b7be-78a4-4e1f-b364-032dc6cacb9c)
![image](https://github.com/user-attachments/assets/9049edc9-6862-4a16-b780-0bf6ea3429b6)
you would just need to make changes to the two url files. but again it's up to you we can make a pr in the future that strictly deals with cleaning up paths